# Apply a force to the center of gravity of a link.
# via the callback mechanism
# force expressed in link's body frame frame.
geometry_msgs/Vector3 force
time start_time                           # (optional) force application start time (seconds)
                                          # if start_time is not specified, or
                                          # start_time < current time, start as soon as possible
duration duration                         # optional duration of force application time (seconds)
                                          # if duration < 0, apply force continuously without end
                                          # if duration = 0, do nothing
                                          # if duration < step size, apply wrench
                                          # for one step size
---
bool success                              # return true if set force successful
string status_message                     # comments if available
